junit-platform-engine/src/main/java/org/junit/platform/engine/support/descriptor/DefaultLegacyReportingInfo.java

```diff
@@ -0,0 +1,33 @@
+package org.junit.platform.engine.support.descriptor;
```

Comment:
Hi @LiamClark, would you kindly run `./gradlew spotlessApply` as soon you're ready, as it will (or, at least should) put an Eclipse Public License header just above this line. :)